Automattic/jetpack
# jetpack: Avoid generating unused JS for static-site-generator assets

#### Changes proposed in this Pull Request:

We use a Webpack static-site-generator plugin to build some HTML files.
This has the side effect of building redundant JS and (in one case) CSS
assets that we don't use.

Let's create a small Webpack plugin to allow us to suppress these unused
assets to reduce the size of the plugin. This may also be of use in the
future if/when we switch the SASS builds to use Webpack instead of gulp.

#### Jetpack product discussion


None directly. We've mentioned needing to build the SASS stuff in a few places.

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Is CI happy?
* Does the Jetpack build no longer contain `_inc/build/static.js`, `_inc/build/static.css`, `_inc/build/static.rtl.css`, or `_inc/blocks/components.js`?
* Am I correct in not finding any uses of any of those files, so they're safe to delete?
